### ISSUE TITLE
Signal version 4.25.7 crashes on backup files from version 4.24.x 

### ISSUE THREAD

**Issue by finnoleary** [signalapp/Signal-Android#8174]
- [x ] I have searched open and closed issues for duplicates
- [ x] I am submitting a bug report for existing functionality that does not work as intended
- [x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Signal Beta version 4.25.7 does not accept backup files from version 4.24.x 

### Steps to reproduce
- Create backup from 4.24.x version
- Install current beta version 4.25.7 on new phone
- Try to import backup from previous version 

**Actual result:** Signal crashes on import of backup
**Expected result:** Signal should import the damn backup

### Device info
Old device:
**Device:** Samsung Galaxy S5
**Android version:** 6.0.1
**Signal version:** v4.24.x

New device:
**Device:** Essential PH-1
**Android version:** 9.0.0
**Signal version:** v4.25.7

### Link to debug log
https://paste.ubuntu.com/p/tghJSVX33s/

**Comment by finnoleary** [comment]
TL;DR: Old version of signal backup does not have the `notification_channel` column. This regression was introduced in commit e9b85a10a6b66c4 and seems to have been present since v4.27.3. Does your dev team not have tests for current - beta version interop?

**Comment by greyson-signal** [comment]
Thanks for catching this! This was accidentally introduced in 3da1a3e2702a8cbd4952f23395f31e1525b462c9 when trying to fix a different backup-related bug. It's times like this that I'm fortunate for our beta testers. Thanks for being one!
